### DISCUSSION TITLE
Vercel & Github Terms and Conditions: Cancellation Process

### DISCUSSION THREAD

**Discussion by SignappsAccounts** [vercel/community#2371] [Help] [answered]
### Summary

Good day

I trust you are well

Our team has requested that we gather all Terms and Conditions from all our suppliers, in particular the process of cancellation for Vercel & Github.

Kindly provide details of same.

I look forward to hearing from you soon

Thank you

**Comment by amyegan** [comment, marked as answer]
Hello @SignappsAccounts. You can find Vercel's Terms of Service here: https://vercel.com/legal/terms

GitHub has posted their Terms of Service here: https://docs.github.com/en/site-policy/github-terms/github-terms-of-service
